src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,111 @@
+package duke.storage;
+import duke.exception.DukeException;
+import duke.task.*;
```

Comment:
Another very minor detail here also, perhaps you can leave a line between your package and your import statement. I see you do that for most of the other classes so maybe you just forget to do it here :D

Reply:
Yep haha I think I overlooked that :D